MapView/Map/RMTileCache.m

```diff
@@ -423,6 +428,16 @@ - (void)beginBackgroundCacheForTileSource:(id <RMTileSource>)tileSource southWes
     }
 }
 
+- (id<RMTileCache>)databaseCacheAtPath:(NSString *)cachePath
+{
+    RMDatabaseCache *dbCache = [[RMDatabaseCache alloc] initWithDatabase:cachePath];
+    [dbCache setCapacity:LONG_MAX];
```

Comment:
For the record, the purpose of this cache is to provide a single source package, basically a hacky way to create an MBTiles package for mapbox sources.